src/beast/beast/container/detail/aged_ordered_container.h

```diff
@@ -1361,6 +1372,7 @@ aged_ordered_container <IsMulti, IsMap, Key, T, Clock, Compare, Allocator>::
 aged_ordered_container (aged_ordered_container&& other,
     Allocator const& alloc)
     : m_config (std::move (other.m_config), alloc)
+    , m_cont (std::move(other.m_cont.comp()))
 {
     insert (other.cbegin(), other.cend());
     other.clear ();
```

Comment:
Do the constructors around lines 1398 and 1422 also need an `,m_cont (comp)`?  Example is here: https://github.com/scottschurr/rippled/commit/4fa293a320ad3efaf06cef07c9fb31d18b386168.